src/lib/components/dispensary-card/dispensary-card.svelte

```diff
@@ -10,9 +10,11 @@
     }
 
     export let dispensary: Dispensary
+    let className: string
+    export { className as class }
```

Comment:
This is a funky pattern that apparently needs to be done to expose `class` as a prop.  See https://github.com/sveltejs/svelte/issues/2870 for discussion... Svelte is not so svelte in some ways.